packages/hash/api/src/task-execution/index.ts

```diff
@@ -0,0 +1,36 @@
+import { DataSource } from "apollo-datasource";
+import fetch from "node-fetch";
+
+// TODO: When task scheduling is more mature and we move away from the temporary `hash-task-executor` we should have a single source of
```

Comment:
Super nit-pick: we've generally tried to use `@todo` in the backend code so far
```suggestion
/** @todo: When task scheduling is more mature and we move away from the temporary `hash-task-executor` we should have a single source of */
```